_includes/manuals/nightly/2.1_quickstart_installation.md

```diff
@@ -82,14 +83,26 @@ sudo yum -y install https://yum.theforeman.org/releases/{{page.version}}/el7/x86
 {% endhighlight %}
 </div>
 
-<div class="quickstart_os quickstart_os_el7">
+<div class="quickstart_os quickstart_os_centos7">
   <p>Enable the EPEL (Extra Packages for Enterprise Linux), Foreman and SCL (Software Collections) repositories:</p>
 
 {% highlight bash %}
-sudo yum -y install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+sudo yum-config-manager --enable extras
```

Comment:
Is this really needed? I've never done this since it's enabled by default. It also gives the impression we rely on extras but it's only really for `epel-release`.

Reply:
IMHO not needed, yeah